### DISCUSSION TITLE
Every Ranked Game Is Garbage

### DISCUSSION THREAD

**Discussion by cedricchi** [ValveSoftware/Dota2-Gameplay#21522] [Show and tell]
Win trade, map hack Pudge, feeding bots, jungle bots, grief, farming supports, jungle pos 3, jungle pos 2, jungle pos 1 not even smelling lane creeps, stomps, stomps, stomps, stomps, stomps, stomps, stomps, stomps

**Comment by JiSinTktank** [comment]
yeah my game meet it :8224336791 

https://steamcommunity.com/id/tmcdg/

meet him just rpoert
